rwengine/src/engine/GameWorld.cpp

```diff
@@ -998,6 +998,9 @@ VehicleObject* GameWorld::tryToSpawnVehicle(VehicleGenerator& gen)
 		/// @todo use zone information to decide vehicle id
 	}
 
+	/// @todo is this sufficient? should dimensions be used?
+	position.z += 1.f;
```

Comment:
Where does the 1.0f come from?
Maybe the game always sends a ray downwards to find the proper spot? What about spawning on slopes?

(Either add more comments or, probably better, make sure to open an issue about these things and explain where values come from)